contrib/fonttools/showttf.c

```diff
@@ -2887,6 +2887,7 @@ static void gdefshowligcaretlist(FILE *ttf,int offset,struct ttfinfo *info) {
 		printf( "!!!! Unknown caret format %d !!!!\n", format );
 	    }
 	}
+        free(offsets);
```

Comment:
I cloned your repository to check. Line 2890 has leading spaces. The preceding lines have tabs. I generally just copy and paste the leading whitespace from an adjacent line when adding something.